File: go/problem1.go
package main

import (
    "fmt" 
)


func main(){

    var x, multiples_sum int

    for x < 1000 {
        if(x%5==0)||(x%3==0){
            fmt.Printf("Multiple of 5 or 3 found: %d\n",x)
            multiples_sum += x
        }
        x += 1

    }
    fmt.Printf("Total of all multiples: %d\n",multiples_sum)

}

File: rust/problem1.rs
fn main() {
    
    let upper_bound = 1000;
    let multiples = vec![3,5];

    let mut x = 0;
    let mut multiples_sum: i32 = 0; //Initialize empty number.
    
    while x < upper_bound {
        
        

    
        if (x%5==0)||(x%3==0) {

            
            println!("Multiple of 5/3 found: {}",x);
            multiples_sum = multiples_sum + x;
            
        
        

        }
        

        x = x+1;
    }
    println!("Total value of multiples found: {}",multiples_sum);
}
